src/libs/ReportUtils.ts

```diff
@@ -4938,14 +4951,14 @@ function isGroupChatAdmin(report: OnyxEntry<Report>, accountID: number) {
 }
 
 /**
- * Helper method to define what money request options we want to show for particular method.
- * There are 4 money request options: Request, Split, Send and Track expense:
- * - Request option should show for:
+ * Helper method to define what expense options we want to show for particular method.
+ * There are 4 expense options: Submit, Split, Pay and Track expense:
+ * - Submit option should show for:
  *     - DMs
  *     - own policy expense chats
  *     - open and processing expense reports tied to own policy expense chat
  *     - unsettled IOU reports
- * - Send option should show for:
+ * - Pay option should show for:
  *     - DMs
  * - Split options should show for:
```

Comment:
Should the split show in goroups where there are 2 and more? i think the comment says 3